tun/tun_darwin.go

```diff
@@ -213,49 +213,50 @@ func (tun *NativeTun) File() *os.File {
 	return tun.tunFile
 }
 
-func (tun *NativeTun) Events() chan Event {
+func (tun *NativeTun) Events() <-chan Event {
 	return tun.events
 }
 
-func (tun *NativeTun) Read(buff []byte, offset int) (int, error) {
+func (tun *NativeTun) Read(buffs [][]byte, sizes []int, offset int) (int, error) {
+	// TODO: the BSDs look very similar in Read() and Write(). They should be
+	//  collapsed, with platform-specific files containing the varying parts of
```

Comment:
I've not seen this two space thing. Is that because `TODO: ` has special significance in your IDE, and the additional space lets the IDE resolve the multiline content as belonging to the TODO: opener? Or is it just a mistake?

Reply:
Yes, multiline editor highlighting related. Not consistent across editors and in function docstrings gofmt will fight it. Removed in https://github.com/WireGuard/wireguard-go/pull/64/commits/8185a1d413db6ba0c07310b3896e5b853405f03c.

Reply:
It's fine if you want to keep it. Just wasn't sure. Either way works for me.